src/components/ContentHighlights/data/constants.js

```diff
@@ -306,3 +330,68 @@ export const TEST_COURSE_HIGHLIGHTS_DATA = [
     ],
   },
 ];
+
+export const testCourseData = [
+  {
+    aggregationKey: 'course:HarvardX+CS50x',
+    title: 'CS50s Introduction to Computer Science',
+    contentType: 'course',
+    partners: [
+      {
+        name: 'Harvard University',
+        logoImageUrl: 'https://prod-discovery.edx-cdn.org/organization/logos/44022f13-20df-4666-9111-cede3e5dc5b6-2cc39992c67a.png',
+      },
+    ],
+    cardImageUrl: 'https://prod-discovery.edx-cdn.org/media/course/image/da1b2400-322b-459b-97b0-0c557f05d017-3b9fb73b5d5d.small.jpg',
+    originalImageUrl: 'https://prod-discovery.edx-cdn.org/media/course/image/da1b2400-322b-459b-97b0-0c557f05d017-a3d1899c3344.png',
+    firstEnrollablePaidSeatPrice: 149,
+  },
+  {
+    aggregationKey: 'course:HarvardX+CS50P',
+    title: 'CS50s Introduction to Programming with Python',
+    contentType: 'course',
+    partners: [
+      {
+        name: 'Harvard University',
+        logoImageUrl: 'https://prod-discovery.edx-cdn.org/organization/logos/44022f13-20df-4666-9111-cede3e5dc5b6-2cc39992c67a.png',
+      },
+    ],
+    originalImageUrl: 'https://prod-discovery.edx-cdn.org/media/course/image/2cc794d0-316d-42f7-bbfd-25c34e4cd5df-033e46d516c0.png',
+    firstEnrollablePaidSeatPrice: 200,
+  },
+  {
+    aggregationKey: 'course:HarvardX+CS50W',
+    title: 'CS50s Web Programming with Python and JavaScript',
+    contentType: 'course',
+    partners: [
+      {
+        name: 'Harvard University',
+        logoImageUrl: 'https://prod-discovery.edx-cdn.org/organization/logos/44022f13-20df-4666-9111-cede3e5dc5b6-2cc39992c67a.png',
+      },
+    ],
+    cardImageUrl: 'https://prod-discovery.edx-cdn.org/media/course/image/8f8e5124-1dab-47e6-8fa6-3fbdc0738f0a-762af069070e.small.jpg',
+    originalImageUrl: 'https://prod-discovery.edx-cdn.org/media/course/image/8f8e5124-1dab-47e6-8fa6-3fbdc0738f0a-4978ad93b1c3.png',
+    firstEnrollablePaidSeatPrice: 249,
+  },
+  {
+    aggregationKey: 'course:HarvardX+CS50AI',
+    title: 'CS50s Introduction to Artificial Intelligence with Python',
+    contentType: 'course',
+    partners: [
+      {
+        name: 'Harvard University',
+        logoImageUrl: 'https://prod-discovery.edx-cdn.org/organization/logos/44022f13-20df-4666-9111-cede3e5dc5b6-2cc39992c67a.png',
+      },
+    ],
+    cardImageUrl: 'https://prod-discovery.edx-cdn.org/media/course/image/3a31db71-de8f-45f1-ae65-11981ed9d680-31634d40b3bb.small.png',
+    originalImageUrl: 'https://prod-discovery.edx-cdn.org/media/course/image/3a31db71-de8f-45f1-ae65-11981ed9d680-b801bb328333.png',
+    firstEnrollablePaidSeatPrice: 199,
+  },
+];
+
+export const testCourseAggregation = {
```

Comment:
[clarification] Is this only used by tests?

Reply:
Yep, this represents the selectedRowIds between state changes of the stepper.